sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/CopyableFileSmbPropertiesList.java

```diff
@@ -90,6 +90,20 @@ public CopyableFileSmbPropertiesList setChangedOn(Boolean changedOn) {
         return this;
     }
 
+    /**
+     * @return whether all properties should be copied from the source file.
+     */
+    public Boolean isAll() {
+        return isSetFileAttributes && isSetCreatedOn && isSetLastWrittenOn && isSetChangedOn;
```

Comment:
Since all these are `Boolean` properties, this will throw NPE if any of them is `null`. The constructor does initialize them but the user can still call `setLastWrittenOn(null)` to make this property `null`. So, we should check for null here.